src/porepy/fracs/fracture_network_2d.py

```diff
@@ -87,16 +82,9 @@ def __init__(
 
         """
 
-        if isinstance(domain, np.ndarray):
-            domain = pp.bounding_box.from_points(domain)
-
-        self.domain: dict[str, float] | None = domain
-        """The domain for this fracture network.
-
-        The domain is defined by a dictionary with keys 'xmin', 'xmax', 'ymin', 'ymax'.
-        If not specified, the domain will be set to the bounding box of the fractures.
-
-        """
+        # TODO: Is it possible to always set pp.Domain as attribute?
```

Comment:
My instinct is no, since we (perhaps?) link the presence of a domain to decide if we do a DFN or DFM-style mesh. However, if that is true, we should perhaps do a new refactoring, but that is for another PR.

Reply:
Fair enough. I think this actually makes a lot of sense. A fracture network without a domain is then simply the fracture set.